content/modules/testing/pages/getting-started.adoc

```diff
@@ -0,0 +1,82 @@
+= Getting Started with Testing
+
+This section contains a practical guide on getting started with creating and running the first test case with Jmix.
+
+By default, every Jmix project already contains the directory structure for automated tests. It also scaffolds one automated test case `UserTest` in the directory: `src/test/java/\<<project-base-package>>/user/UserTest.java` that acts as an example for to create your own test classes.
```

Comment:
Let's assume this guide talks about v.1.5 and classic UI. So there are 2 tests in a new project created with "Full-stack application (Java)" template: UserTest and UserUiTest.